src/capitalsFirst.js

```diff
@@ -21,7 +21,12 @@
  * @returns {string} - string with uppercase words in front
  */
 function capitalsFirst(str) {
-  // write code here
+  const wordsArray = str.split(' ');
+  let upperWords = wordsArray.filter(word => word[0] >= 'A' && word[0] <= 'Z');
+  let lowerWords = wordsArray.filter(word => word[0] >= 'a' && word[0] <= 'z');
```

Comment:
you don\`t need let if you don\`t reassign a variable [prefer-const rule](https://mate-academy.github.io/style-guides/javascript.html#variables--const)